### PR TITLE
[NETBEANS-54] Module Review versioning

### LINKED ISSUE #207: Cleanup NetBeans license header for module versioning
Review and cleanup the license header in versioning

Oracle donated NetBeans to the Apache Software Foundation.

Your task now is to *replace* the old license header that has Oracle (and/or Sun Microsystems
and/or NetBeans) in it with the standard Apache Software Foundation license header.

For an empty file that supports headers you must *add* the Apache license header.

Note that you have to use the proper license header for Java, .properties or HTML files.
Look at the nearby files for templates and copy-paste the license header.

When done commit the changes with a commit message that starts with first line
"[NETBEANS-54] Module Review <module>" and a message body describing your review.

See a longer explanation about what we are doing here: https://cwiki.apache.org/confluence/display/NETBEANS/List+of+Modules+to+Review

Please announce in a comment when you have started work.

Files that need to be looked at in versioning:
```
versioning/test/qa-functional/data/tck.properties.template
versioning/test/unit/data/workdir/root-test-versioned/a.txt
versioning/test/unit/data/workdir/root-test-versioned/b-test-versioned/b.txt
versioning/test/unit/data/workdir/root-with-exclusions/folder/excl/newfile
versioning/test/unit/data/workdir/root-with-exclusions/folder/newfile
versioning/test/unit/data/workdir/root-with-exclusions/newfile
versioning/test/unit/data/workdir/root-with-exclusions/subfolder-excl/file
versioning/test/unit/data/workdir/root/a.txt

```

### PR DESCRIPTION
**Pull request by trevlewis**
Fix #207

Update old Oracle license to Apache License.

### CODE REVIEW

File: versioning/apichanges.xml [hunk 1]
```diff
@@ -108,7 +82,7 @@ is the proper place.
     <!-- ACTUAL CHANGES BEGIN HERE: -->
 
     <changes>
-        
+
```
Comment (emilianbold) [inline versioning/apichanges.xml]: Don't change whitespace.

File: versioning/apichanges.xml [hunk 2]
```diff
@@ -1,48 +1,22 @@
...
-Version 2 license, then the option applies only if the new code is
-made subject to such option by the copyright holder.
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
```
Comment (emilianbold) [inline versioning/apichanges.xml]: Doesn't look like the other xml headers (indenting, etc.) Just copy-paste from somewhere existing.

File: versioning/apichanges.xml [hunk 3]
```diff
@@ -146,12 +120,12 @@ is the proper place.
             <author login="tstupka"/>
             <compatibility addition="yes"/>
             <description>
-                New method <code>VCSHistoryProvider.HistoryEntry.getParent()</code> to provide the 
+                New method <code>VCSHistoryProvider.HistoryEntry.getParent()</code> to provide the
```
Comment (emilianbold) [inline versioning/apichanges.xml]: More space at the end.

File: versioning/apichanges.xml [hunk 4]
```diff
@@ -322,7 +296,7 @@ is the proper place.
                 a chance to check-out a file in the beforeEdit() callback.
             </description>
         </change>
-        
+
```
Comment (emilianbold) [inline versioning/apichanges.xml]: Whitespace.

**Review by emilianbold — CHANGES_REQUESTED** [review]

**Comment by trevlewis** [comment]
The license for versioning/apichanges.xml was added by another PR.

**Review by emilianbold — APPROVED** [review]

**Comment by emilianbold** [comment]
Could you fix the conflict? Looks good otherwise.

**Comment by trevlewis** [comment]
@emilianbold Good to go now.

**Pull request merged**